addons/survey/models/survey_question.py

```diff
@@ -268,6 +269,18 @@ def _compute_save_as_email(self):
             if question.question_type != 'char_box' or not question.validation_email:
                 question.save_as_email = False
 
+    @api.depends('survey_id.session_speed_rating_timer', 'survey_id.session_speed_rating', 'is_time_limited')
+    def _compute_time_limit(self):
+        for question in self:
+            if not question.is_time_limited:
+                if question.survey_id.session_speed_rating:
+                    question.time_limit = question.survey_id.session_speed_rating_timer or 0
+                elif question.time_limit == question.survey_id.session_speed_rating_timer:
+                    question.time_limit = False
+
```

Comment:
Let us say I create a question, that does not go in the if nor the elif, then I try to read its time_limit. Does it return 0 or is this an issue? :) This could be discussed at review ^^ Maybe I'm being too cautious

Reply:
From what I tested and how I envision the process to go: 
1) If you create a question and user has speed rating enabled, then session_speed_rating_time_limit will be true and the question will get a default value. 
2) If the user does not have speed rating enabled, it will indeed have no value stored. However, once user enables speed rating, the question will get a default value at that moment unless user already specified custom time limit on the question beforehand